package/MDAnalysis/analysis/bat.py

```diff
@@ -500,6 +501,7 @@ def Cartesian(self, bat_frame):
         bonds = bat_frame[9:n_torsions + 9]
         angles = bat_frame[n_torsions + 9:2 * n_torsions + 9]
         torsions = bat_frame[2 * n_torsions + 9:]
+        copy.deepcopy()
```

Comment:
A few issues here:

1) you don't actually provide an argument to `deepcopy()`, so it isn't going to copy anything
2) you don't capture the return value, so even if you did deepcopy something, it wouldn't do anything useful in the control flow
3) There is no regression test for the issue--you could use something like `np.may_share_memory()` to check for input/output pass by reference issues for example.

Reply:
Hey, According to third point as you said i need to add a regression test , So can you suggest in which file should i add the test regarding it. As i am a beginner i dont know the codebase, it would be great if you could help me with it. Thanks!

Reply:
The tests for that modules are here: <https://github.com/MDAnalysis/mdanalysis/blob/develop/testsuite/MDAnalysisTests/analysis/test_bat.py>. That would be a good place for your tests.